shared/macros-bash.jinja

```diff
@@ -1165,12 +1179,8 @@ if test -L "{{{ config_file }}}"; then
     sed_command+=('--follow-symlinks')
 fi
 
-# If the cce arg is empty, CCE is not assigned.
-if [ -z "{{{ cce }}}" ]; then
-    cce="CCE"
-else
-    cce="{{{ cce }}}"
-fi
+# Set the rule CCE value; if the rule has none, it is set to 'CCE'
+{{{ set_cce_value() }}}
```

Comment:
I think that now when we don't use the XSLT we have an opportunity to make the ` bash_replace_or_append` Jinja macro less stupid or even more intelligent.

Currently,  when the rule doesn't have any CCE number assigned, it prints `Per CCE: Set something in somefile.`. However, this feels like a weird workaround for the situtation when there is no CCE. If there is no CCE, it doesn't make sense to talk about CCEs at all. In my opinion, when a rule has no CCE we shouldn't put this this comment at all. What do you think?